content/docs/lists-and-keys.md

```diff
@@ -202,13 +212,16 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
+[**Probeer het op CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
 
-A good rule of thumb is that elements inside the `map()` call need keys.
+Een goede vuistregel is dat de elementen binnen een `map()` aanroep keys nodig hebben.
 
+### Keys Moeten Uniek Zijn Ten Opzichte Van Siblings {#keys-must-only-be-unique-among-siblings}
```

Comment:
```suggestion
### Keys Moeten Uniek Zijn Ten Opzichte van Naasten {#keys-must-only-be-unique-among-siblings}
```

We don't have a direct translation for `siblings`, but in this case `naasten` might work.